### PR TITLE
User types doc

### PR DESCRIPTION
**Pull request by mgwalker**
Converts the doc from #179 into Markdown so it can live inside the repo.  It's at `docs/users-and-roles.md`.  Going to take @NAretakis to review and approve this one - feel free to ping me or @brendansudol if you want to pair up for the review!

### This pull request is ready to merge when...
- ~~Tests have been updated (and all tests are passing)~~
- [x] This code has been reviewed by someone other than the original author
- ~~The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~~
- [x] The change has been documented
  - ~~Associated OpenAPI documentation has been updated~~

### This feature is done when...
- ~~Design has approved the experience~~
- [x] Product has approved the experience - cc: @NAretakis

### CODE REVIEW

File: docs/users-and-roles.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+essentially a collection of activities.
+
+### Roles
+
+The system has some predefined roles. New roles can be added as the system
+is running, and the predefined roles can be modified. These are the roles
+as they are defined at the "beginning":
+
+| Name               | Activities                                                                                                                  | Description                                                                                                                                                                                                    |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (NAretakis) [inline docs/users-and-roles.md]: @mgwalker Can we expand activities column so that it formats a little nicer?

Comment (mgwalker) [reply docs/users-and-roles.md]: Unfortunately, this is a function of how Github displays Markdown tables.  I'll poke around with it, but as far as I know, there's not a lot we can do to change the formatting.

Comment (NAretakis) [reply docs/users-and-roles.md]: got it, no problem. I had to try and find something for my first one.

Comment (mgwalker) [reply docs/users-and-roles.md]: Give it a look now, let me know what you think.  😄

**Pull request merged**